### DISCUSSION TITLE
Cannot delete all selected chats in chat list

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#2098] [General]
Hi. When selecting many chats in Chat List and clicking on Delete selected, not all chats are deleted. It seems it can only delete max 20 chats or so.  So If I set to 100 items per page and select all it does not delete them all.
Has this been fixed recently?

**Comment by remdex** [comment]
Just fixed in this commit https://github.com/LiveHelperChat/livehelperchat/commit/700a232319ea8b6cc89c5c9d9ce9242e26a5a880

**Comment by alonsoac** [comment]
Thanks!
